pdf-workitem/pom.xml

```diff
@@ -0,0 +1,199 @@
+<?xml version="1.0"?>
+<project
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+    xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jbpm.contrib</groupId>
+    <artifactId>workitems</artifactId>
+    <version>7.12.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>pdf-workitem</artifactId>
+  <name>PDF</name>
+  <description>Create PDF from Thymeleaf template</description>
+  <organization>
+    <name>jBPM dev team</name>
+  </organization>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.thymeleaf</groupId>
+      <artifactId>thymeleaf</artifactId>
```

Comment:
Can we use freemarker for templates instead as we already have it on classpth of kie server/jbpm? And by that make it scope provided

Reply:
Can I ask why? It's not causing any issues in the workbench and has more features than freemarker or stringtemplate 4 imo.

Reply:
mainly to avoid too many libraries that do the same thing. Do you use these features that free marker does not have?

Reply:
for simpler examples no, but i think if user want some complex things it would make it easier, but im sure still doable with freemarker so i will replace.